src/core/utils/misc.cpp

```diff
@@ -553,7 +553,33 @@ void Utils::Misc::openFolderSelect(const QString& absolutePath)
     if (QFileInfo(path).exists()) {
         // Syntax is: explorer /select, "C:\Folder1\Folder2\file_to_select"
         // Dir separators MUST be win-style slashes
-        QProcess::startDetached("explorer.exe", QStringList() << "/select," << Utils::Fs::toNativePath(absolutePath));
+
+        // QProcess::startDetached() has an obscure bug. If the path has
+        // no spaces and a comma(and maybe other special characters) it doesn't
```

Comment:
Not really obscure. explorer.exe is doing the unusual thing here, that is using commas as command switch separator. To prevent commas from being considered switches, you must escape them and escaping is done by wrapping with double quotes (I've just found that it's enough to wrap just the comma).

startDetached() will correctly wrap args with spaces. I should say this is correct as long as args are supplied as QStringList and it's correct to assume that each element of the list is actually a single argument, which doesn't seem to be the case on Windows.